doc/man1/openssl-dgst.pod.in

```diff
@@ -86,7 +86,15 @@ Output the digest or signature in binary form.
 
 =item B<-xoflen> I<length>
 
-Set the output length for XOF algorithms, such as B<shake128>.
+Set the output length for XOF algorithms, such as B<shake128> and B<shake256>.
+This option is not supported for signing operations.
+It is recommended to set this value to at least 32 to ensure a security strength
+of 128 bits.
+
+B<shake128> defaults to 16 (which is only a security strength of 64 bits).
```

Comment:
It's unclear to me which security level 128 bit of shake128 actually provides. But you need at least 256 bit of output to reach the 128 bit security level.

Reply:
Have updated the text.